skyrl-train/skyrl_train/generators/skyrl_gym_generator.py

```
@@ -34,10 +35,27 @@ def __init__(
         self.skyrl_gym_cfg = skyrl_gym_cfg
         self.inference_engine_client = inference_engine_client
         self.tokenizer = tokenizer
+        self.model_name = model_name
         self.max_turns = generator_cfg.max_turns
         self.batched = generator_cfg.batched
         self.use_conversation_multi_turn = generator_cfg.use_conversation_multi_turn
 
+        self.use_http_server_inference_engine_client = generator_cfg.get(
+            "use_http_server_inference_engine_client", False
+        )
+        self.http_server_inference_engine_client_host = generator_cfg.get(
+            "http_server_inference_engine_client_host", "127.0.0.1"
+        )
+        self.http_server_inference_engine_client_port = generator_cfg.get(
+            "http_server_inference_engine_client_port", 8000
+        )
+
+        if self.use_http_server_inference_engine_client:
+            # Store the base URL for direct HTTP requests
+            self.base_url = f"http://{self.http_server_inference_engine_client_host}:{self.http_server_inference_engine_client_port}"
+        else:
+            self.base_url = None
+
```

Is `use_conversation_multi_turn` supported with the http server? I think not, because we'd need a `/completions` endpoint that supports token in, token out. 

Can you raise an error here if `use_conversation_multi_turn=False` with http server?